budcamp/Homework3/Homework3/Lithuania.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework3
+{
+    internal class Lithuania: History, ICountry
+    {
+        public int Population { get; set; }
+        public string Gymn { get; set; }
+        public string Flag { get; set; }
+
+        public Lithuania(int population, string gymn, string flag, string deadRussians) : base(deadRussians)
+        {
+            this.Population = population;
+            this.Gymn = gymn;
+            this.Flag = flag;
+        }
+
+        private string Weapon()
```

Comment:
It is a common convention that method names should be verbs or start with verbs. 
Please change the name to `GetWeapon` or `AskForWeapon`